conjureup/consts.py

```diff
@@ -4,6 +4,7 @@
 JAAS_CLOUDS = {'ec2', 'azure', 'gce'}
 JAAS_DOMAIN = 'jimm.jujucharms.com'
 JAAS_ENDPOINT = JAAS_DOMAIN + ':443'
+PROVIDER_TYPES = ['localhost', 'maas', 'vsphere', 'openstack']
```

Comment:
I'm not sure this is the best name, since it is a bit confusing with the list of Provider models.  Maybe something like `CUSTOM_CLOUDS`?